src/qt/mAPreferencesWindow.cpp

```diff
@@ -83,15 +89,76 @@ const QString mAPreferencesInputChannels = "/VM/InputChannels";
 const QString mAPreferencesSampleRate = "/VM/SampleRate";
 const QString mAPreferencesBufferSize = "/VM/SampleBufferSize";
 const QString mAPreferencesVMStallTimeout = "/VM/StallTimeout";
+const QString mAPreferencesAudioDriver = "/VM/AudioDriver"; // 1.5.0.1 (ge) added
 
 const QString mAPreferencesEnableNetwork = "/VM/EnableNetworkOTFCommands";
 const QString mAPreferencesEnableAudio = "/VM/EnableAudio";
 
 
+// global symbols expected from RtAudio | 1.5.0.1 (ge) added
+extern "C" const unsigned int rtaudio_num_compiled_apis;
+extern "C" const unsigned int rtaudio_compiled_apis[];
+extern "C" const char * rtaudio_api_names[][2];
```

Comment:
seems preferable to have this accessed via `ChuckAudio::` functions

Reply:
done